### PR TITLE
[34070] Improve board creation modal

### PR DESCRIPTION
**Pull request by bsatarnejad**
adding images, change the text and styles of  tile component

https://community.openproject.com/projects/openproject/work_packages/34070/activity

### CODE REVIEW

**Comment by openprojectci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/c0ab0cc7decd0d94a3f73bb2e7ff1715a9e52808/frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts#L42">frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts#L42</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts [hunk 1]
```diff
@@ -37,7 +37,9 @@ import {BoardService} from "core-app/modules/boards/board/board.service";
 import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-actions/board-actions-registry.service";
 import {LoadingIndicatorService} from "core-app/modules/common/loading-indicator/loading-indicator.service";
 import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-resource-notification.service";
-import { ITileViewEntry } from '../tile-view/tile-view.component';
+import {ITileViewEntry} from '../tile-view/tile-view.component';
+import {ImageHelpers} from "core-app/helpers/images/path-helper";
+import imagePath = ImageHelpers.imagePath;
```
Comment (openprojectci) [inline frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts:44]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/37f390b16231302b56c62f84b478eb99a32f9f33/frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts#L44"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

**Comment by oliverguenther** [comment]
Hi @bsatarnejad , the image paths were corrected. You can now continue

**Review by oliverguenther — APPROVED** [review]
LGTM :+1:

**Pull request merged**